linera-core/src/chain_worker/state.rs

```diff
@@ -277,8 +274,11 @@ where
             .verify_owner(&proposal)
             .ok_or(WorkerError::InvalidOwner)?;
         proposal.check_signature(public_key)?;
-        // Check the authentication of the operations in the block.
-        if let Some(signer) = block.authenticated_signer {
+        if let Some(validated) = validated {
```

Comment:
Unrelated: the name `validated` does not follow the coding guidelines (adjective instead of a noun). As a matter of fact I keep forgetting what it is (and the field is not documented in `BlockProposal`). Is this a `ValidationCertificate` ?

Reply:
It's a `Certificate` with a `ValidatedBlock` value, yes. Maybe `validated_block_certificate`?